packages/flutter/lib/src/widgets/two_pane.dart

```diff
@@ -0,0 +1,482 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show DisplayFeature;
+
+import 'basic.dart';
+import 'debug.dart';
+import 'framework.dart';
+import 'media_query.dart';
+
+/// A widget that positions two panes side by side on uninterrupted screens or
+/// on either side of a separating [DisplayFeature] on screens interrupted by a
+/// separating [DisplayFeature].
+///
+/// A [DisplayFeature] separates the screen into sub-screens when both these
+/// conditions are met:
+///
+///   * it obstructs the screen, meaning the area it occupies is not 0. Display
+///     features of type [DisplayFeatureType.fold] can have height 0 or width 0
+///     and not be obstructing the screen.
```

Comment:
Does this mean the TwoPane will not work with devices that fold without a hinge?

Reply:
It means it works, but the separation between the two panes does not need to coincide with the folding feature since the screen is continuous.